extensions/ql-vscode/src/remote-queries/variant-analysis-view.ts

```diff
@@ -130,4 +134,29 @@ export class VariantAnalysisView extends AbstractWebview<ToVariantAnalysisMessag
       void showAndLogWarningMessage(`Could not open file: ${variantAnalysis.query.filePath}`);
     }
   }
+
+  private async openQueryText(): Promise<void> {
+    const variantAnalysis = await this.manager.getVariantAnalysis(this.variantAnalysisId);
+    if (!variantAnalysis) {
+      void showAndLogWarningMessage('Could not open variant analysis query text');
+      return;
+    }
+
+    const filename = variantAnalysis.query.filePath;
+
+    try {
+      const params = new URLSearchParams({
+        variantAnalysisId: variantAnalysis.id.toString(),
+      });
+      const uri = Uri.from({
+        scheme: 'codeql-variant-analysis',
+        path: filename,
```

Comment:
Is this path used for anything? I can't see it being read anywhere in`createVariantAnalysisContentProvider`. I suppose a URI needs to have a path, but I was wondering if it needs to be the query file path or if it could be a fixed path.

Reply:
It seems like VSCode uses this path for displaying the filename, so that's why I made it match the filename of the query. Remote queries will always use a fixed filename (`query-text.ql`), but I think it might be useful to also show the actual query filename.

Reply:
Ah, makes sense. Thanks for explaining.